config/ember.js

```diff
@@ -1,9 +1,18 @@
 module.exports = {
-  extends: require.resolve('./base.js'),
+  extends: [
+    'plugin:ember/recommended',
+    require.resolve('./base.js')
+  ],
   env: {
     'browser': true
   },
   rules: {
-    'new-cap': [ 'error', { 'capIsNewExceptions': [ 'A' ] } ]
+    'new-cap': [ 'error', { 'capIsNewExceptions': [ 'A' ] } ],
+
+    // Custom rules
+    'ember/avoid-leaking-state-in-components': 'error',
+    'ember/local-modules': 'error',
+    'ember/no-empty-attrs': 'error',
+    'ember/no-global-jquery': 'off'
```

Comment:
Why are you turning this rule off? Seems like a useful one.

Reply:
My original thought behind this is that we still have applications that require importing the jQuery module. An example would be in Calculon for grabbing the report's HTML to export to our PDF service.

But now that I think about it, I guess we can leave the default setting of `'error'` and explicitly disable the ESLint rule where we need to import the jQuery module in.

Reply:
That's not what the rule is meant to catch, at least according to my reading. The rule is meant to catch using jquery as a true global, like without importing or destructuring it. So

```javascript
import $ from 'jquery';
```

or

```javascript
import Ember from 'ember';

const { $ } = Ember;
```

should prevent it from being triggered, but if neither of those are present, it should trigger the rule. Did you try it out and see something different?

Reply:
Hmmm... Okay, this is interesting then. When I linked these changes into Calculon, it was complaining about `import $ from 'jquery';` as it being a global in the `report-page` component. Let me revisit this and I'll get back to you with my findings.

Reply:
Hmm...looks like it might be a bug. The [tests](https://github.com/ember-cli/eslint-plugin-ember/blob/master/tests/lib/rules/no-global-jquery.js) for the rule don't include any tests for the new module import syntax, so maybe they just overlooked that case?

Reply:
You beat me to it. Yes, I think they did. On my free time, I can update this rule and submit a PR. Would you like me to turn this rule off in the mean time until I can submit a fix/update for this rule?

Reply:
Yeah, let's leave it off until it's fixed upstream